vertigo-bot-designer/src/main/javagen/io/vertigo/chatbot/designer/builder/BuilderPAO.java

```diff
@@ -58,6 +58,50 @@ public void cleanOldTrainings(@io.vertigo.datamodel.task.proxy.TaskInput(name =
 		getTaskManager().execute(task);
 	}
 
+	/**
+	 * Execute la tache TkGetAllReponseTypeIdsByBotId.
+	 * @param botId Long
+	 * @return List de Long filIds
+	*/
+	@io.vertigo.datamodel.task.proxy.TaskAnnotation(

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.